tools/picard/picard_CollectAlignmentSummaryMetrics.xml

```diff
@@ -24,7 +24,9 @@
     #for $sequence in $adapters:
         ADAPTER_SEQUENCE="${sequence.adapter}"
     #end for
-    METRIC_ACCUMULATION_LEVEL="${metric_accumulation_level}"
+    #for $level in ${metric_accumulation_level}:
```

Comment:
Hi @bgruening, this does not work for me (galaxy@15.07). 
*Long-form placeholders - ${}, $(), $[], etc. are not valid inside expressions.* ==> should be ``in $metric_accumulation_level``. 
But changing that results in:
``TypeError: 'SelectToolParameterWrapper' object is not iterable``